modules/heat_transfer/test/tests/sp3_radiation/adaptive1D_e1_segregated.i

```diff
@@ -0,0 +1,778 @@
+n1 = 1.46
+n2 = 1.00
+
+alpha = 0.914
+alpha1 =  0.084075389781548382
+alpha2 =  0.872433935441263020
+beta1 = -0.08086509944859642
+beta2 = -0.29284032175563224
+eta1 =  -2.6234175064678195
+eta2 = 9.9471473598607409
+
+k = 1
+h = 0.001
+Tb = 300
+T0 = 1000
+epsilon = 0.1
+endt = 0.0001
+dt = 2.5e-5
+
+nu_min = 1e-2
+nu1 = 2.93e13
+nu2 = 3.42e13
+nu3 = 3.73e13
+nu4 = 4.56e13
+nu5 = 5.13e13
+nu6 = 5.87e13
+nu7 = 6.84e13
+nu8 = 102.67e13
+nu_max = 1e16
+
+kappa1to2 = 7136.00
+kappa2to3 = 576.32
+kappa3to4 = 276.98
+kappa4to5 = 27.98
+kappa5to6 = 15.45
+kappa6to7 = 7.7
+kappa7to8 = 0.5
+kappa8tomax = 0.4
+
+[Mesh]
+  [gmg]
+      type = GeneratedMeshGenerator
+      dim = 1
+      nx = 51
+      xmin = -0.5
+      xmax = 0.5
+  []
+[]
+
+[Variables]
+  [T]
+    type = MooseVariableFVReal
+    initial_condition = ${T0}
+  []
```

Comment:
since all variables are actually in the same nonlinear system, this is the fully coupled approach not the segregated.

this is the test I wanted to see, just change the name / description please